### ISSUE TITLE
Tabbing in - GroupedList basic and example-app-base

### ISSUE THREAD

**Issue by kadio1** [OfficeDev/office-ui-fabric-react#12127]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: chrome and edge

### Describe the issue:

The tabbing structure is a bit off. I believe you should be able to close and expand each collapsible with enter or space bar. This should also work if the master collapsible under the column name has been invoked. If the items are not selectable, they should not have the grey hover state either. 

#### Actual behavior:

Enter selects the row as oppose to opens or closes. 

#### Expected behavior:

should be able to open and close a group.

### Documentation describing expected behavior

![tabbing02](https://user-images.githubusercontent.com/41807199/75589434-95b74480-5a2f-11ea-946f-fcce2e8c8da6.PNG)
![tabbing01](https://user-images.githubusercontent.com/41807199/75589435-964fdb00-5a2f-11ea-98b5-eea850c12131.PNG)

**Comment by aneeshack4** [comment]
Hi @kadio1, thanks for reaching out to us! I believe there's a keyboard shortcut besides enter & space for collapse and expand - asking around to find out what this is & will report back.

**Comment by aneeshack4** [comment]
Looks like it used be the right/left arrow but not sure how far back/if ever this was working as it's been open issue for a while #6318 - there's an open PR addressing this #11494 which is currently waiting on more feedback. I'll close this as it's being tracked in the original issue and in the PR.
